docs/release-notes.mdx

```diff
@@ -8,6 +8,12 @@ import { TitleCard } from './components/TitleCard';
     Keep up to date with the latest releases of the design system.
 </TitleCard>
 
+## Unreleased
```

Comment:
We don't need to put this into release because it is technical documentation, not public